opc-ua-stack/stack-core/src/main/java/org/eclipse/milo/opcua/stack/core/Stack.java

```diff
@@ -105,7 +120,7 @@ public Thread newThread(@Nonnull Runnable r) {
                 }
             };
 
-            SCHEDULED_EXECUTOR_SERVICE = Executors.newSingleThreadScheduledExecutor(threadFactory);
+            SCHEDULED_EXECUTOR_SERVICE = Executors.newScheduledThreadPool(5);
```

Comment:
Any reason you chose 5 instead of something like `Runtime.getRuntime().availableProcessors()`?

And can you use the overload that also accepts the `threadFactory`